### PR TITLE
Status end point and graylog implementation

### PR DESCRIPTION
**Pull request by flaredragon**
- Status End Point Added For OneAuth at \status to check service of postgres

### CODE REVIEW

File: routers/statusrouter.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const router = require('express').Router();
+
+router.get('/', function(req, res, next) {
+    var seq=require('../db/models.js');
```
Comment (witty123) [inline routers/statusrouter.js]: require only once

**Comment by witty123** [comment]
@flaredragon the PR has some merge conflicts, can you get them resolved ?

**Comment by flaredragon** [comment]
Done The Changes.
@witty123 Please Review.

File: server.js [hunk 2]
```diff
@@ -23,7 +23,9 @@ const config = require('./config')
     , apirouter = require('./routers/api')
     , oauthrouter = require('./routers/oauthrouter')
     , pagerouter = require('./routers/pagerouter')
+    , statusrouter = require('./routers/statusrouter')
     , {expresstracer, datadogRouter} = require('./utils/ddtracer')
+    , {ExpressLogger} = require('./utils/logger');
```
Comment (witty123) [inline server.js]: Follow standard naming convention please

File: utils/logger.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+const ExpressLogger = ExpressWinston.logger ({
+  transports: [GrayLogger],
+  meta: true,
+  msg: "HTTP {{req.method}} {{req.url}}",
+  expressFormat: true,
+  colorize: false,
+  ignoreRoute:function (req,res) {
+    if(req.method==='GET'&&req.url.indexOf("css") > -1)
+    return true;
+    if(req.method==='GET'&&req.url.indexOf("js") > -1)
```
Comment (witty123) [inline utils/logger.js]: why three different if blocks ?

File: utils/logger.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+})
+
+const ExpressLogger = ExpressWinston.logger ({
+  transports: [GrayLogger],
+  meta: true,
+  msg: "HTTP {{req.method}} {{req.url}}",
+  expressFormat: true,
+  colorize: false,
+  ignoreRoute:function (req,res) {
+    if(req.method==='GET'&&req.url.indexOf("css") > -1)
```
Comment (witty123) [inline utils/logger.js]: ideally static content should be served through nginx only and wouldnt reach node app server, we can remove these statements

**Comment by flaredragon** [comment]
Sir I'll Add Graylog As another Pull Request with the required changes there were some major merge conflicts due to that.

File: routers/statusrouter.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    res.send({
+    	postgres: 'Connected'
+    })
+	})
+   .catch(err => {
+    res.send({
+    	postgres: 'Not Connected'
+    })
+	});
+   });
```
Comment (championswimmer) [inline routers/statusrouter.js]: basic formatting issues

File: server.js [hunk 6]
```diff
@@ -53,6 +55,7 @@ app.engine('hbs', exphbs.express4({
 app.set('views', path.join(__dirname, 'views'));
 app.set("view engine", "hbs");
 
+app.use (expressLogger);
```
Comment (championswimmer) [inline server.js]: no space

File: server.js [hunk 7]
```diff
@@ -79,6 +82,7 @@ app.use('/signup', signuprouter);
 app.use('/api', apirouter);
 app.use('/oauth', oauthrouter);
 app.use('/', pagerouter);
+app.use('/status',statusrouter);
```
Comment (championswimmer) [inline server.js]: space after comma

**Comment by witty123** [comment]
@championswimmer merge and deploy on priority please. Need to put up monitoring on all the services today.

**Pull request merged**